sql-parser/src/main/antlr/SqlBase.g4

```diff
@@ -40,7 +40,7 @@ statement
     | UPDATE aliasedRelation
         SET assignment (',' assignment)*
         where?
-        (RETURNING selectItem (',' selectItem)*)?      													 	   #update
+        returning?      								                             #update
```

Comment:
The indentation here looks way off?

Reply:
Would
```UPDATE aliasedRelation SET assignment (',' assignment)* where? returning?```
then way to go ?

Reply:
Ah sorry should have been more specific. It was related to the extra whitespace before the `#update` (all the others were aligned)